tests/db_functions/tests.py

```diff
@@ -25,6 +25,49 @@ def truncate_microseconds(value):
 
 class FunctionTests(TestCase):
 
+    def test_strindex(self):
+        Author.objects.create(name='G. R. R. Martin')
+        Author.objects.create(name='J. R. R. Tolkien')
+        Author.objects.create(name='Terry Pratchett')
+        authors = Author.objects.annotate(fullstop=StrIndex('name', '.'))
```

Comment:
Maybe it will be better to use `'R.'` instead of `'.'` in tests. IMO it can be misleading with regexp `'.'`, moreover it will be nice to test string rather then single character.